test-utils/runtime-tester/src/lib.rs

```diff
@@ -5,8 +5,10 @@
 //! fuzzing provides Arbitrary trait for Scenario, thus enabling creating random scenarios.
 pub mod fuzzing;
 pub mod run_test;
+pub mod scenario_builder;
 
 pub use crate::run_test::{BlockConfig, NetworkConfig, Scenario, TransactionConfig};
+pub use crate::scenario_builder::ScenarioBuilder;
```

Comment:
This is a minor code smell -- now the caller can use `ScenarioBuilder` as either `runtime_tested::ScenarioBuilder` or `runtime_tester::scenario_builer::ScenarioBuilder`, and it's not immediately clear from the code which variant is preferred. 

I usually prefer to make API with only one possilbe import path. For smaller crates, it's usually best for the caller if all the API is available at root, and all the internals are private. 

But that's a minor point, the code looks ok as is!